src/App.js

```diff
@@ -19,36 +20,7 @@ function App() {
   return (
     <div className="App">
       <header className="App-header sticky" id="header">
-        <ul>
```

Comment:
Why is this being deleted?

Reply:
Oops will revert this, this was deleted coz this block was obstructing the view of other pages during my build stage.